rpc-client-api/Cargo.toml

```diff
@@ -12,7 +12,7 @@ edition = { workspace = true }
 [dependencies]
 anyhow = { workspace = true }
 base64 = { workspace = true }
-bs58 = { workspace = true }
+bs58 = { workspace = true, features = ["std"] }
```

Comment:
It's definitely ok to have this, but just to check, this needs `std` because of std::error::Error, correct? https://github.com/Nullus157/bs58-rs/blob/e4a65a9ec641f26b61fb20518b51e83efd6352a0/src/lib.rs#L38

Reply:
correct